packages/application-shell/package.json

```diff
@@ -86,7 +87,8 @@
     "redux-thunk": "2.3.0",
     "reselect": "3.0.1",
     "unfetch": "3.1.1",
-    "uuid": "3.3.2"
+    "uuid": "3.3.2",
+    "warning": "4.0.2"
```

Comment:
this is used [here](https://github.com/commercetools/merchant-center-application-kit/blob/4bdb1197f26d03f3ce77ed33d5d7b47db93e4d41/packages/application-shell/src/from-core/deprecate-component/deprecate-component.js)